pyproject.toml

```
@@ -0,0 +1,14 @@
+[build-system]
+requires = [
+  # Essentials
+  "setuptools>=40.6.0",
+  "wheel",
+]
```

Without these, there's no configuration to include files into dists and to set any version:
```suggestion

  # Plugins
  "setuptools_scm[toml]>=3.5",
  "setuptools_scm_git_archive>=1.1",
]
```